docs/content/technical-specifications.md

```diff
@@ -12,7 +12,8 @@ docs: "DOCS-617"
 
 We advise users to run the most recent release of the NGINX Ingress Controller, and we issue software updates to the most recent release. We provide technical support for F5 customers who are using the most recent version of the NGINX Ingress Controller, and any version released within two years of the current release.
 
-The current release version 3.x supports `discovery.k8s.io/v1` API version of EndpointSlice, available from Kubernetes 1.21 onwards.
+The 3.1.0 release removes `NET_BIND_SERVICE` from binary files' capability sets, and the manifests set `net.ipv4.ip_unprivileged_port_start` sysctl to allow port binding. This requires the sysctl to be [classified as safe](https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#safe-and-unsafe-sysctls), which is done in Kubernetes 1.22 and later.
```

Comment:
Not sure here - this probably should be a major bump of `4.0.0`. Let me know if this should be tweaked.

Also, the table versions will need to be further tweaked to properly represent the release.

Reply:
I don't think this should be a major bump (`4.x`), we're just removing a version of k8s that is EOL anyway, as you said, and using new functionalities that are in the new supported versions. I don't see it as breaking change tbh.

Reply:
With the change, it would be breaking for users who are staying on an old image tag. Not sure why they would be specifying an old image tag, but who knows. If that's an okay change, then no need for `4.x`.